ethereum_history_api/circuits/lib/src/account_test.nr

```diff
@@ -0,0 +1,21 @@
+mod account_within_block_serialize {
+    use crate::account_with_storage::ACCOUNT_SERIALIZED_LEN;
+    use crate::account::{AccountWithinBlock, ACCOUNT_WITHIN_BLOCK_SERIALIZED_LEN};
+    use crate::fixtures::mainnet::london::vitalik_balance::{account::account, header::hash};
+    use crate::misc::arrays::sub_array_equals;
+
+    #[test]
+    fn success() {
```

Comment:
To consider: change test name from success to simple, as success seems as if there could be some failing test